.github/workflows/std.yml

```diff
@@ -95,100 +110,131 @@ jobs:
+
+            echo ==================================================
+
+            aws configure list
+
+            aws sts get-caller-identity
+
+            kubectl cluster-info -n dev-preview
+
+            echo ==================================================
+
+            ### FIXME: change to dev-preview!!!
+
+            nix run -L ".#cardano-services.dev-preprod@us-east-1.plan" | tee k8s-plan.diff
+          '
+      - name: Post Comment on the PR
         env:
-          BRANCH: ${{ github.ref_type == 'branch' && github.head_ref }}
           GH_TOKEN: ${{ github.token }}
-          OWNER_AND_REPO: ${{ github.repository }}
-        with: {ffBuildInstructions: true, remoteStore: "ssh-ng://eu.nixbuild.net"}
+        run: |
+          prNumber=$(cut -d/ -f1 <<<'${{ github.ref_name }}')
+          gh pr comment "$prNumber" --body "$(sed -e '2i\\n```diff' -e '$a```' k8s-plan.diff)"
 
 
-  deploy-to-eu:
-    runs-on: ubuntu-latest
-    needs: [discover, images]
-    name: ${{ matrix.target.jobName }} (us-east-1)
-    env:
-      AWS_REGION: us-east-1
-    permissions:
-      id-token: write
-      contents: read
-      deployments: write
+  deploy-to-us:
+    if: github.event_name == 'push' && github.ref_name == 'dev-preview'
+    needs: [images]
+    concurrency:
+      # Only one deployment at a time per environment, and wait for the previous one to finish:
+      group: deploy-${{ github.ref_name }}
+      cancel-in-progress: false
+    name: Deploy (${{ github.ref_name }})
+    runs-on: ubuntu-22.04
     environment:
-      name: dev-preview
-      url: https://backend.dev-preview.eks.lw.iog.io
-    # Boolean input should be compared with string until https://github.com/actions/runner/issues/2238 resolved
-    if: >
-      fromJSON(needs.discover.outputs.hits).deployments.apply != '{}' && ((github.event_name == 'push' && github.ref == 'refs/heads/master') || inputs.deploy)
-
-    strategy:
-      matrix:
-        target: ${{ fromJSON(needs.discover.outputs.hits).deployments.apply }}
+      name: ${{ github.ref_name }}
+      url: https://${{ github.ref_name }}.dapps.lw.iog.io/
     steps:
-      - name: Configure AWS Credentials
-        uses: aws-actions/configure-aws-credentials@v2.2.0
-        with:
-          role-to-assume: ${{ env.AWS_ROLE_ARN }}
-          aws-region: ${{ env.AWS_REGION }}
+      - uses: actions/checkout@v4
       - uses: nixbuild/nix-quick-install-action@v25
       - uses: nixbuild/nixbuild-action@v17
         with:
           nixbuild_ssh_key: ${{ secrets.SSH_PRIVATE_KEY }}
           generate_summary_for: job
+      # Further steps assume AWS_PROFILE=lw, while the official action has no way to specify that profile:
+      - name: Set up AWS credentials
+        run: |
+          mkdir -p ~/.aws
+
+          cat <<EOF >~/.aws/credentials
+          [lw]
+          aws_access_key_id = ${{ secrets.AWS_ACCESS_KEY}}
+          aws_secret_access_key = ${{ secrets.AWS_SECRET_ACCESS_KEY }}
```

Comment:
Arent env variables enough?

Reply:
I’ll use them, okay

Reply:
Fixed in d1c671b5b0ae79783fec28ad11d3d76ef63d75da

Reply:
I think [env. variables are not enough](https://github.com/input-output-hk/cardano-js-sdk/actions/runs/8139418959/job/22242557797?pr=1119) (same error was happening in Dapps). 

```
Error: Failed to get release dev-preprod-cardanojs in namespace dev-preprod: exit status 1: 
The config profile (lw) could not be found
Error: Kubernetes cluster unreachable: Get "https://eb65a880b90df0a9b8455038ef27a45e.gr7.us-east-1.eks.amazonaws.com/version": getting credentials: exec: executable aws failed with exit code 255
```

The `lw` part in the INI file seems crucial :thinking:

Reply:
Let me revert d1c671b5b0ae79783fec28ad11d3d76ef63d75da…

Reply:
And now it works again :green_circle: